android/app/src/main/java/com/gpslogger/LocationModule.java

```diff
@@ -66,6 +66,7 @@ public String createLocationJSON(Location location) throws JSONException {
         JSONObject json = new JSONObject();
         json.put("longitude", location.getLongitude());
         json.put("latitude", location.getLatitude());
+	json.put("altitude", location.getAltitude());
```

Comment:
Fixed in https://github.com/jochem725/reactnative-gps-logger/pull/7/commits/6eb2d4a1a971c687fafca16da37b495f1a54c14a